pallets/operations/src/lib.rs

```diff
@@ -0,0 +1,84 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![doc = include_str!("../README.md")]
+
+extern crate core;
+
+mod impls;
+mod traits;
+
+#[cfg(test)]
+mod tests;
+
+use frame_support::traits::{LockIdentifier, StorageVersion};
+
+const STORAGE_VERSION: StorageVersion = StorageVersion::new(0);
+pub(crate) const LOG_TARGET: &str = "pallet-operations";
+// harcoding as those consts are not public in substrate
+pub(crate) const STAKING_ID: LockIdentifier = *b"staking ";
+pub(crate) const VESTING_ID: LockIdentifier = *b"vesting ";
```

Comment:
just a nit nothing important, use either pub or const here (without (crate) so timorl will be happy)

Reply:
Okay